nemo_skills/dataset/hle/prepare.py

```diff
@@ -26,16 +27,16 @@ def format_entry(entry):
         "answer_type": entry["answer_type"],
         "solution": entry["rationale"],
```

Comment:
Let's rename for consistency
```suggestion
        "reference_solution": entry["rationale"],
```